risczero/Cargo.toml

```diff
@@ -3,28 +3,51 @@ name = "risczero-benchmark"
 version = "0.1.0"
 edition = "2021"
 
+[workspace]
+members = [
+    "methods",
+]
+
+[workspace.dependencies]
+risczero-benchmark-methods = { path = "methods" }
+
+[workspace.dependencies.risc0-build]
+tag = "v0.13.0"
+git = "https://github.com/risc0/risc0.git"
```

Comment:
Since the crates are published, I think we should prefer to use `version = 0.13` and drop the `git` value.